tests/sentry/workflow_engine/models/test_data_condition.py

```diff
@@ -4,6 +4,36 @@
 from sentry.workflow_engine.types import DetectorPriorityLevel
 
 
+class GetConditionResultTest(TestCase):
+    def test_str(self):
+        dc = self.create_data_condition(condition_result="wrong")
+        with mock.patch("sentry.workflow_engine.models.data_condition.logger") as mock_logger:
+            assert dc.get_condition_result() is None
+            assert mock_logger.error.call_args[0][0] == "Invalid condition result"
+
+    def test_int(self):
+        dc = self.create_data_condition(condition_result=1)
+        assert dc.get_condition_result() == 1
+
+    def test_float(self):
+        dc = self.create_data_condition(condition_result=1.0)
+        assert dc.get_condition_result() == 1.0
+
+    def test_int__overlaps_with_priority_low(self):
+        dc = self.create_data_condition(condition_result=25)
+        assert dc.get_condition_result() == 25
+        assert dc.get_condition_result() == DetectorPriorityLevel.LOW
```

Comment:
I think this passes because `DetectorPriorityLevel` is an `IntEnum` and python is automatically converting the result to an int for comparison

Reply:
it 100% does, but i wanted to illustrate that it won't be a huge issue if the int collides with a the priority level because of duck typing. should we not allow ints and force floats?